src/pretalx/orga/templates/orga/cfp/text.html

```diff
@@ -166,6 +154,26 @@
                         </div>
                     </fieldset>
 
+                    <fieldset>
+                        <legend id="profile">{% translate "Speaker Profile" %}</legend>
+                        <div class="offset-md-3 alert alert-info"><span>
+                            {% translate "Speaker Profile" %}
+                        </span></div>
+                        <div class="table-responsive-md"><table class="table offset-md-3 table-sm table-hover table-flip" id="cfp-option-table">
+                                <tr>
+                                    <th>{% translate "Biography" %}</th>
+                                    <td class="hide-label">{% bootstrap_field sform.cfp_ask_biography layout='event-inline' %}</td>
+                                    <td>{% bootstrap_field sform.cfp_biography_min_length use_label=False layout='inline' %}</td>
+                                    <td>{% bootstrap_field sform.cfp_biography_max_length use_label=False layout='inline' %}</td>
+                                </tr>
+                                <tr>
+                                    <th>{% translate "Profile picture" %}</th>
+                                    <td class="hide-label">{% bootstrap_field sform.cfp_ask_avatar layout='event-inline' %}</td>
+                                    <td></td>
+                                    <td></td>
+                                </tr>
+		        </table></div>
```

Comment:
each fieldset contains a table. this closing table on line 175 gets opened in line 162

Reply:
Indeed, but please keep indentation in a way that this can be seen. The line 162 has the div and the table, but alignment with line 175 is off.
Probably better to have div and table on different lines with increasing indentation, both on opening and closing.

Reply:
this is exactly how the original structure was where i copied this from. when working on an existing codebase, i prefer to duplicate the style that already exists instead of confusing things by introducing a new style. 

you may argue that this is an opportunity to clean up, but as i am new to the project, doing that potentially introduces a different style than the one that we actually want, so i prefer to wait with cleanup until i am familiar with the desired style.

also if i change this now, i would have to go and change it for the whole file which introduces changes to lines that otherwise do not need to be changed, making the PR harder to read. or i would end up introducing a second style into this file. both are things i an not a fan of.

i value style consistency because that helps readability of the code and keeping style changes separate from logic changes to help readability of commits.

Reply:
Sorry missed the table tag, still looking at https://github.com/fossasia/eventyay-talk/blob/main/src/pretalx/orga/templates/orga/cfp/text.html it seems the table and div should be aligned with the div and table opening, so I'll suggest:

```suggestion
                        </table></div>
```